pinax/api/jsonapi.py

```diff
@@ -43,7 +46,7 @@ def from_validation_error(cls, exc, resource_class):
                 errs.append(err)
         return cls(errors=errs)
 
-    def __init__(self, data=None, errors=None, links=False, included=None, meta=None, linkage=False):
+    def __init__(self, data=None, errors=None, links=False, included=None, meta={}, linkage=False):
```

Comment:
This is an ugly no-no in Python. The dictionary (if used) is scoped globally and can have bad side-affects to meta across requests (it can share values across all requests).